src/Kopernicus/Templates.cs

```diff
@@ -52,6 +52,9 @@ public class Templates
         // The body that should appear in MainMenu
         public static String menuBody { get; set; }
 
+        // The main menu color
+        public static ColorParser mainMenuLightColor = new ColorParser(new Color(1, 1, 1, 1));
```

Comment:
You should just use a Color here, not a color parser.

Also, it would be awesome if the color in the main menu could be automatically fetched from the star the main menu body is orbiting. I am not sure if Kopernicus already has an API for that (probably `KopernicusStar.GetNearest(MainMenuBody).lightShifter.lightColor` or so), if not I can write down the exact code later.